prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -231,6 +231,25 @@ function generateAndExportLetsEncryptCert() {
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
+    #encrypt the tls cert
+	gcloud kms encrypt --location global \
+	--keyring "${KYMA_KEYRING}" \
+	--key "${KYMA_ENCRYPTION_KEY}"
+	--plaintext-file ./letsencrypt/live/"${DOMAIN}"/fullchain.pem  \
+	--ciphertext-file "nightly-aks-tls-integration-app-client-cert.encrypted"
+	
+	#encrypt the private cert
+	gcloud kms encrypt --location global \
+	--keyring "${KYMA_KEYRING}" \
+	--key "${KYMA_ENCRYPTION_KEY}"
+	--plaintext-file ./letsencrypt/live/"${DOMAIN}"/fullchain.pem  \
```

Comment:
it looks that you encrypt twice the same file.

Reply:
updated in both gke and aks